_docs-sources/pipelines/architecture/index.md

```diff
@@ -0,0 +1,45 @@
+# Architecture
+
+## Overview
+
+There are three major components in the Gruntwork Pipelines architecture:
+
+1. The orchestrator
+2. The dispatcher
+3. The executor
```

Comment:
Could we simplify our architecture to just the orchestrator and the executor? For example, if the orchestrator does the dispatching to the executor, do we need to call out the dispatcher as a separate component?

Reply:
Maybe, but for now we want to describe what exists.

Reply:
That is the plan eventually. We don't have that work scheduled for now.